python-backend/app/api/mines/compliance/models/compliance_article.py

```diff
@@ -13,8 +13,14 @@ class ComplianceArticle(AuditMixin, Base):
     paragraph = db.Column(db.String, nullable=True)
     sub_paragraph = db.Column(db.String, nullable=True)
     description = db.Column(db.String, nullable=False)
+    long_description = db.Column(db.String, nullable=False)
     effective_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
-    expiry_date = db.Column(db.DateTime, nullable=False, default=datetime.strptime('9999-12-31', '%Y-%m-%d'))
+    expiry_date = db.Column(
+        db.DateTime, nullable=False, default=datetime.strptime('9999-12-31', '%Y-%m-%d'))
```

Comment:
i think this is the default that @NWCalvank is trying to hunt down. This is now a nullable column with no default right?

Reply:
It's not. I missed this table while trying to find all of the 9999-12-31 columns.

I updated: mine_location, permit_status_code, party_type_code, party, mineral_tenure_xref, mine_status, mine_region_code, and article_act_code. But I missed compliance_article, so the db default for it is still 9999-12-31